# French translations for minarca_client.
# Copyright (C) 2021 ORGANIZATION
# This file is distributed under the same license as the minarca_client
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
#
msgid ""
msgstr ""
"Project-Id-Version: minarca_client 3.9.1.dev11+g4d5317c\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2023-07-12 15:14-0400\n"
"PO-Revision-Date: 2023-07-12 15:19-0400\n"
"Last-Translator: \n"
"Language-Team: fr <LL@li.org>\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
"Generated-By: Babel 2.10.3\n"
"X-Generator: Poedit 3.0.1\n"

#: minarca_client/main.py:42
msgid "Operation aborted by the user."
msgstr "Opération interrompue par l'utilisateur."

#: minarca_client/main.py:51
msgid "yes"
msgstr "oui"

#: minarca_client/main.py:51
msgid "y"
msgstr "o"

#: minarca_client/main.py:60
#, python-format
msgid "new version %s available"
msgstr "nouvelle version % s disponible"

#: minarca_client/main.py:62
msgid "fail to check for latest version"
msgstr "impossible de vérifier pour la dernière version"

#: minarca_client/main.py:82
msgid "minarca is already linked, execute `minarca unlink`"
msgstr "minarca est déjà lié, exécutez `minarca unlink`"

#: minarca_client/main.py:89
msgid "password or access token: "
msgstr "mot de passe ou jeton d'accès: "

#: minarca_client/main.py:96 minarca_client/main.py:101
#, python-format
msgid "Linked successfully"
msgstr "Lié avec succès"

#: minarca_client/main.py:99 minarca_client/ui/setup.py:147
msgid "Do you want to replace the existing repository ?"
msgstr "Vous souhaitez remplacer le référentiel existant ?"

#: minarca_client/main.py:152
msgid ""
"Are you sure you want to proceed with the restore operation? Please note that "
"this will override your current data. (Yes/No): "
msgstr ""
"Êtes-vous sûr de vouloir procéder à l'opération de restauration ? Veuillez "
"noter que cette opération annulera vos données actuelles. (Oui/Non) : "

#: minarca_client/main.py:167
#, python-format
msgid "Do you want to restore %s (Yes/No): "
msgstr "Voulez-vous restaurer %s (Oui/Non) : "

#: minarca_client/main.py:194
msgid "backup not running"
msgstr "la sauvegarde ne s'exécute pas"

#: minarca_client/main.py:227
#, python-format
msgid "Remote server:          %s"
msgstr "Serveur distant :       %s"

#: minarca_client/main.py:228
#, python-format
msgid "Connectivity status:    %s"
msgstr "État de la connectivité : %s"

#: minarca_client/main.py:229
#, python-format
msgid "Last successful backup: %s"
msgstr "Dernière sauvegarde réussie : %s"

#: minarca_client/main.py:229 minarca_client/main.py:230
#: minarca_client/main.py:231
msgid "Never"
msgstr "Jamais"

#: minarca_client/main.py:230
#, python-format
msgid "Last backup date:       %s"
msgstr "Date de la dernière sauvegarde : %s"

#: minarca_client/main.py:231
#, python-format
msgid "Last backup status:     %s"
msgstr "État de la dernière sauvegarde : %s"

#: minarca_client/main.py:232
#, python-format
msgid "Details:                %s"
msgstr "Détails:                %s"

#: minarca_client/main.py:234
#, python-format
msgid "Paused until:           %s"
msgstr "En pause jusqu'à :           %s"

#: minarca_client/main.py:261
msgid ""
"Minarca manage your computer's backup by linking your computer with a "
"centralized server and running backups on a given schedule."
msgstr ""
"Minarca gère la sauvegarde de votre ordinateur en reliant votre ordinateur à un "
"serveur centralisé et en exécutant des sauvegardes selon un calendrier donné."

#: minarca_client/main.py:280
msgid "start a backup in background mode"
msgstr "démarrer une sauvegarde en arrière-plan"

#: minarca_client/main.py:281 minarca_client/main.py:286
msgid "force execution of a backup even if it's not time to run"
msgstr ""
"forcer l'exécution d'une sauvegarde même s'il n'est pas temps de l'exécuter"

#: minarca_client/main.py:285
msgid "start a backup in foreground mode"
msgstr "démarrer une sauvegarde en mode avant-plan"

#: minarca_client/main.py:290
msgid "exclude files to be backup"
msgstr "exclure les fichiers à sauvegarder"

#: minarca_client/main.py:291 minarca_client/main.py:297
msgid "file pattern to be exclude. may contains `*` or `?` wildcard"
msgstr ""
"modèle de fichier à exclure. peut contenir un caractère générique `*` ou `?`"

#: minarca_client/main.py:296
msgid "include files to be backup"
msgstr "inclure les fichiers à sauvegarder"

#: minarca_client/main.py:302
msgid "link this minarca backup with a minarca server"
msgstr "lier cette sauvegarde minarca avec un serveur minarca"

#: minarca_client/main.py:303
msgid "URL to the remote minarca server. e.g.: http://example.com:8080/"
msgstr "URL vers le serveur minarca distant. p.e.: http://example.com:8080/"

#: minarca_client/main.py:304
msgid "user name to be used for authentication"
msgstr "nom d'utilisateur à utiliser pour l'authentification"

#: minarca_client/main.py:306
msgid ""
"password or access token to use for authentication. Will prompt if not provided"
msgstr ""
"mot de passe ou jeton d'accès à utiliser pour l'authentification. Il serait "
"demandé s'il n'est pas fourni"

#: minarca_client/main.py:308
msgid "repository name to be used"
msgstr "nom du dépôt à utiliser"

#: minarca_client/main.py:310
msgid "link to remote server even if the repository name already exists"
msgstr "lier avec le serveur distant même si le nom du référentiel existe déjà"

#: minarca_client/main.py:315
msgid "list the includes / excludes patterns"
msgstr "liste les modèles d'inclusion/exclusion"

#: minarca_client/main.py:319
msgid "restore data from backup"
msgstr "restaurer des données à partir d'une sauvegarde"

#: minarca_client/main.py:322
msgid ""
"Date time to be restored. Could be 'now' to retore the latest backup. Could be "
"an epoch value like '1682367069'. Could be an ISO date format like "
"'2023-02-24T04:11:09-04:00'. Could be an interval like '3D' for 3 days ago."
msgstr ""
"Date et heure de la restauration. Il peut s'agir de \"now\" pour restaurer la "
"dernière sauvegarde. Il peut s'agir d'une valeur d'époque comme '1682367069'. "
"Il peut s'agir d'un format de date ISO comme '2023-02-24T04:11:09-04:00'. Il "
"peut s'agir d'un intervalle comme \"3D\" pour 3 jours."

#: minarca_client/main.py:327
msgid "force execution of restore operation without confirmation from user"
msgstr ""
"forcer l'exécution d'une opération de restauration sans confirmation de "
"l'utilisateur"

#: minarca_client/main.py:329
msgid "files and folders to be restore"
msgstr "fichiers et dossiers à restaurer"

#: minarca_client/main.py:333
msgid "stop the backup"
msgstr "arrêter la sauvegarde"

#: minarca_client/main.py:334
msgid "doesn't fail if the backup is not running"
msgstr "n'échoue pas si la sauvegarde n'est pas en cours d'exécution"

#: minarca_client/main.py:339
msgid "create required schedule task in crontab or Windows Task Scheduler"
msgstr ""
"créer la tâche de planification requise dans crontab ou le planificateur de "
"tâches Windows"

#: minarca_client/main.py:346
msgid "schedule backup to run hourly"
msgstr "planifier la sauvegarde pour qu'elle s'exécute toutes les heures"

#: minarca_client/main.py:353
msgid "schedule backup to run daily"
msgstr "planifier la sauvegarde pour qu'elle s'exécute quotidiennement"

#: minarca_client/main.py:360
msgid "schedule backup to run weekly"
msgstr "planifier la sauvegarde pour qu'elle s'exécute chaque semaine"

#: minarca_client/main.py:363
msgid "username required to run task when user is logged out"
msgstr ""
"nom d'utilisateur requis pour exécuter la tâche lorsque l'utilisateur est "
"déconnecté"

#: minarca_client/main.py:364
msgid "password required to run task when user is logged out"
msgstr ""
"mot de passe requis pour exécuter la tâche lorsque l'utilisateur est déconnecté"

#: minarca_client/main.py:368
msgid "return the current minarca status"
msgstr "retourner l'état actuel de la minarca"

#: minarca_client/main.py:372
msgid "unlink this minarca client from server"
msgstr "dissocier ce client minarca du serveur"

#: minarca_client/main.py:378
msgid ""
"temporarily delay the execution of the backup for the given amount of hours. "
"Default 24 hours."
msgstr ""
"retarde temporairement l'exécution de la sauvegarde pendant le nombre d'heures "
"indiqué. Valeur par défaut : 24 heures."

#: minarca_client/main.py:380
msgid "number of hours"
msgstr "nombre d'heures"

#: minarca_client/main.py:385
msgid "clear previously set delay"
msgstr "efface le délai précédemment défini"

#: minarca_client/main.py:394
msgid "open graphical user interface (default when calling minarcaw)"
msgstr ""
"ouvrir l'interface utilisateur graphique (par défaut avec l'execution de "
"minarcaw)"

#: minarca_client/core/__init__.py:436
msgid "generating identity"
msgstr "création d'une identité"

#: minarca_client/core/__init__.py:442
msgid "exchanging identity with minarca server"
msgstr "échange d'identité avec le serveur Minarca"

#: minarca_client/core/__init__.py:446
msgid "generating new identity"
msgstr "création d'une nouvelle identité"

#: minarca_client/core/__init__.py:450
msgid "exchanging new identity with minarca server"
msgstr "échange de la nouvelle identité avec le serveur Minarca"

#: minarca_client/core/__init__.py:506
#, python-format
msgid "executing command: %s"
msgstr "exécution de la commande : %s"

#: minarca_client/core/compat.py:283 minarca_client/ui/templates/home.html:1
#: minarca_client/ui/templates/setup.html:1
msgid "Minarca Backup"
msgstr "Sauvegarde Minarca"

#: minarca_client/core/compat.py:338
msgid ""
"Keeps your Minarca Backup Software running. If this task is disabled or "
"stopped, your Minarca backup will stop working. This task gets created when "
"linking Minarca with a central backup server. When installing Minarca on a "
"Windows Server, it's preferable to change the settings of this task and select "
"'Run whether user is logged on or not' to make sure the backup is running even "
"when nobody is using the server."
msgstr ""
"Maintient le fonctionnement de votre logiciel de sauvegarde Minarca. Si cette "
"tâche est désactivée ou arrêté, votre sauvegarde Minarca cessera de "
"fonctionner. Cette tâche est créée lors de la liaison de Minarca avec un "
"serveur de sauvegarde central. Lors de l'installation Minarca sur un serveur "
"Windows, il est préférable de modifier les paramètres de cette tâche et "
"sélectionnez « Exécuter que l'utilisateur soit connecté ou non » pour vous "
"assurer la sauvegarde s'exécute même lorsque personne n'utilise le serveur."

#: minarca_client/core/config.py:220
msgid "User's Documents"
msgstr "Documents de l'utilisateur"

#: minarca_client/core/config.py:227
msgid "Thumbnails cache"
msgstr "Cache de vignettes"

#: minarca_client/core/config.py:228
msgid "Arrangement of a Windows folder"
msgstr "Agencement d'un dossier Windows"

#: minarca_client/core/config.py:229
msgid "Swap System File"
msgstr "Fichier d'échange (Mémoire Virtuelle)"

#: minarca_client/core/config.py:230
msgid "Page System File"
msgstr "Fichier de pagination"

#: minarca_client/core/config.py:231
msgid "Hibernation System File"
msgstr "Fichier d'hibernation"

#: minarca_client/core/config.py:232
msgid "System Volume Information"
msgstr "System Volume Information"

#: minarca_client/core/config.py:233
msgid "System Recovery"
msgstr "Récupération système"

#: minarca_client/core/config.py:234
msgid "Recycle bin"
msgstr "Corbeille"

#: minarca_client/core/config.py:235 minarca_client/core/config.py:254
msgid "Temporary Folder"
msgstr "Dossier temporaire"

#: minarca_client/core/config.py:236
msgid "AutoCAD backup files"
msgstr "Fichiers de sauvegarde AutoCAD"

#: minarca_client/core/config.py:237
msgid "Office temporary files"
msgstr "Fichiers temporaires Office"

#: minarca_client/core/config.py:238
msgid "Outlook IMAP temporary files"
msgstr "Fichiers temporaires d'Outlook IMAP"

#: minarca_client/core/config.py:239
msgid "Outlook POP temporary files"
msgstr "Fichiers temporaires d'Outlook POP"

#: minarca_client/core/config.py:245
msgid "Desktop Services Store"
msgstr "Desktop Services Store"

#: minarca_client/core/config.py:251
msgid "dev filesystem"
msgstr "système de fichiers /dev"

#: minarca_client/core/config.py:252
msgid "proc filesystem"
msgstr "système de fichiers /proc"

#: minarca_client/core/config.py:253
msgid "sys filesystem"
msgstr "système de fichiers /sys"

#: minarca_client/core/config.py:255
msgid "Volatile program files"
msgstr "Fichiers volatile /run"

#: minarca_client/core/config.py:256
msgid "Mounted filesystems"
msgstr "Systèmes de fichiers montés /mnt"

#: minarca_client/core/config.py:257
msgid "External media"
msgstr "Médias externes /media"

#: minarca_client/core/config.py:258
msgid "Ext4 Lost and Found"
msgstr "Ext4 perdu et trouvé"

#: minarca_client/core/config.py:259
msgid "Hidden temporary files"
msgstr "Fichiers temporaires cachés"

#: minarca_client/core/config.py:260
msgid "Vim Temporary files"
msgstr "Fichiers temporaires Vim"

#: minarca_client/core/exceptions.py:49
#, python-format
msgid ""
"Fail to link because repository with name `%s` already exists on remote server."
msgstr ""
"Echec de la liaison car le dépôt avec le nom `%s` existe déjà sur le serveur "
"distant."

#: minarca_client/core/exceptions.py:62
#, python-format
msgid ""
"your file specification [%s] doesn't matches any of the base directory of your "
"system"
msgstr ""
"votre spécification de fichier [%s] ne correspond à aucun des répertoires de "
"base de votre système"

#: minarca_client/core/exceptions.py:72
#, python-format
msgid ""
"backup process terminated with an exception, check logs for more details: %s"
msgstr ""
"le processus de sauvegarde s'est terminé avec une exception, vérifiez les "
"journaux pour plus de détails : %s"

#: minarca_client/core/exceptions.py:80
msgid "backup process returned non-zero exit status, check logs for more details"
msgstr ""
"le processus de sauvegarde a renvoyé un état de sortie non nul, consultez les "
"journaux pour plus de détails"

#: minarca_client/core/exceptions.py:88
#: minarca_client/core/tests/test_backup.py:638
msgid "include patterns are missing"
msgstr "les modèles à inclure sont manquants"

#: minarca_client/core/exceptions.py:96
msgid "cannot stop backup when it's not running"
msgstr ""
"impossible d'arrêter la sauvegarde lorsqu'elle n'est pas en cours d'exécution"

#: minarca_client/core/exceptions.py:104
msgid "cannot start backup when it's already running"
msgstr ""
"impossible de démarrer la sauvegarde alors qu'elle est déjà en cours d'exécution"

#: minarca_client/core/exceptions.py:112
msgid "not configured, use `minarca link` to configure remote host"
msgstr "non configuré, utilisez `minarca link` pour configurer l'hôte distant"

#: minarca_client/core/exceptions.py:120
msgid "backup not yet scheduled to run, you may force execution using `--force`"
msgstr ""
"sauvegarde n'est pas encore planifiée pour s'exécuter, vous pouvez forcer "
"l'exécution en utilisant `--force`"

#: minarca_client/core/exceptions.py:130
#, python-format
msgid "cannot establish connection to `%s`, verify if the URL is valid"
msgstr "impossible d'établir la connexion à `%s`, vérifiez si l'URL est valide"

#: minarca_client/core/exceptions.py:141
#, python-format
msgid ""
"the given URL `%s` is not properly formated, verify if the URL is valid. It "
"must start with either https:// or http://"
msgstr ""
"l'URL donnée `%s` n'est pas correctement formatée, vérifiez si l'URL est "
"valide. Il doit commencer par https:// ou http://"

#: minarca_client/core/exceptions.py:153
msgid "authentication refused, verify your username and password"
msgstr ""
"authentification refusée, vérifiez votre nom d'utilisateur et votre mot de passe"

#: minarca_client/core/exceptions.py:161
msgid ""
"remote server return an error, check remote server log with your administrator"
msgstr ""
"le serveur distant renvoie une erreur, vérifiez le journal du serveur distant "
"avec votre administrateur"

#: minarca_client/core/exceptions.py:169
msgid ""
"Unable to connect to the remote server. The problem may be with the remote "
"server. If the problem persists, contact your system administrator to check the "
"SSH server configuration and a possible firewall blocking the connection."
msgstr ""
"Impossible de se connecter au serveur distant. Le problème peut provenir du "
"serveur distant. Si le problème persiste, contactez votre administrateur "
"système pour vérifier la configuration du serveur SSH et un éventuel pare-feu "
"bloquant la connexion."

#: minarca_client/core/exceptions.py:184
msgid ""
"Backup failed due to our identity being refused by remote server. The problem "
"may be with the remote server. If the problem persists, contact your system "
"administrator to review your SSH identity."
msgstr ""
"La sauvegarde a échoué parce que votre identité a été refusée par le serveur "
"distant. Le problème peut provenir du serveur distant. Si le problème persiste, "
"contactez votre administrateur système pour vérifier votre identité SSH."

#: minarca_client/core/exceptions.py:198
msgid ""
"Backup failed due to unknown remote server identity. If the problem persists, "
"contact your system administrator to review the server identity."
msgstr ""
"La sauvegarde a échoué en raison de l'identité inconnue du serveur distant. Si "
"le problème persiste, contactez votre administrateur système pour vérifier "
"l'identité du serveur."

#: minarca_client/core/exceptions.py:212
msgid ""
"Backup failed due to disk quota exceeded. Please free up disk space to ensure "
"successful backup."
msgstr ""
"La sauvegarde a échoué en raison d'un dépassement du quota de disque. Veuillez "
"libérer de l'espace disque pour assurer la réussite de la sauvegarde."

#: minarca_client/core/exceptions.py:224
msgid ""
"Backup failed due disk is full. Please clear space on the disk to proceed with "
"the backup."
msgstr ""
"La sauvegarde a échoué car le disque est plein. Veuillez libérer de l'espace "
"sur le disque pour poursuivre la sauvegarde."

#: minarca_client/core/exceptions.py:236
msgid ""
"Backup failed due unresolvable hostname. Please check your network connection "
"and ensure the hostname is valid."
msgstr ""
"La sauvegarde a échoué en raison d'un nom d'hôte non résoluble. Veuillez "
"vérifier votre connexion réseau et vous assurer que le nom d'hôte est valide."

#: minarca_client/core/exceptions.py:250
msgid ""
"Backup failed due to unsupported Minarca agent version on remote server. "
"Consider upgrading your agent or your server."
msgstr ""
"La sauvegarde a échoué en raison d'une version non prise en charge de l'agent "
"Minarca sur le serveur distant. Pensez à mettre à jour votre agent ou votre "
"serveur."

#: minarca_client/core/latest.py:31
msgid "https://minarca.org/download/"
msgstr "https://minarca.org/download/"

#: minarca_client/ui/patterns.py:28
msgid "Included"
msgstr "Inclus"

#: minarca_client/ui/patterns.py:28
msgid "Excluded"
msgstr "Exclu"

#: minarca_client/ui/patterns.py:80
msgid "Add Folder Pattern"
msgstr "Ajouter modèle"

#: minarca_client/ui/patterns.py:93
msgid "Add custom pattern"
msgstr "Ajouter modèle"

#: minarca_client/ui/patterns.py:94
msgid ""
"You may provide a custom pattern to include or exclude file.\n"
"Custom pattern may include wildcard `*` or `?`."
msgstr ""
"Vous pouvez fournir un modèle personnalisé pour inclure ou exclure le fichier.\n"
"Le modèle personnalisé peut inclure le caractère générique `*` ou `?`."

#: minarca_client/ui/schedule.py:96
msgid "Pause Backup Job Confirmation"
msgstr "Confirmation de la mise en pause de la sauvegarde"

#: minarca_client/ui/schedule.py:97
msgid "Are you sure you want to pause the backup job?"
msgstr "Êtes-vous sûr de vouloir mettre en pause la tâche de sauvegarde ?"

#: minarca_client/ui/schedule.py:98
msgid ""
"Pausing the backup job will temporarily suspend the ongoing backup process for "
"the next 12 hours. This action may affect the availability of recent data "
"restore points and introduce delays in the backup schedule. Please confirm your "
"intention before proceeding."
msgstr ""
"La mise en pause de la tâche de sauvegarde suspend temporairement la sauvegarde "
"pour les 12 heures à venir. Cette action peut affecter la disponibilité des "
"points de restauration des données et entraîner des retards dans le calendrier "
"de sauvegarde. Veuillez confirmer votre intention avant de poursuivre."

#: minarca_client/ui/schedule.py:136 minarca_client/ui/templates/credential.html:1
msgid "Task Scheduler"
msgstr "Planificateur de tâches"

#: minarca_client/ui/schedule.py:137
msgid "Task Scheduler cannot apply your changes."
msgstr "Le planificateur de tâches ne peut pas appliquer vos changements."

#: minarca_client/ui/settings.py:54
msgid "New version available"
msgstr "Nouvelle version disponible"

#: minarca_client/ui/settings.py:55
#, python-format
msgid ""
"A new version of Minarca (%s) is available from IKUS Software. Do you want to "
"upgrade your copy ?"
msgstr ""
"Une nouvelle version de Minarca (%s) est disponible chez IKUS Software. Voulez-"
"vous mettre à jour votre copie ?"

#: minarca_client/ui/settings.py:59
msgid ""
"Minarca automatically checks for new updates. You can change how Minarca checks "
"for updates in Minarca's settings."
msgstr ""
"Minarca recherche automatiquement les nouvelles mises à jour. Vous pouvez "
"modifier la façon dont Minarca vérifie les mises à jour dans les paramètres de "
"Minarca."

#: minarca_client/ui/settings.py:99
msgid "Are you sure ?"
msgstr "Êtes-vous sûr ?"

#: minarca_client/ui/settings.py:100
msgid "Are you sure you want to disconnect this Minarca agent ?"
msgstr "Êtes-vous sûr de vouloir déconnecter cet agent Minarca ?"

#: minarca_client/ui/settings.py:101
msgid ""
"If you disconnect this computer, this Minarca agent will erase its identity and "
"will no longer run backup on schedule."
msgstr ""
"Si vous déconnectez cet ordinateur, cet agent Minarca effacera son identité et "
"n'exécutera plus la sauvegarde."

#: minarca_client/ui/setup.py:81 minarca_client/ui/setup.py:82
msgid "Invalid remote server URL !"
msgstr "URL du serveur distant non valide !"

#: minarca_client/ui/setup.py:83
msgid ""
"The remote server URL you entered for the connection is not valid. Check that "
"you have entered the correct value. The URL must begin with `http://` or "
"`https://` followed by a domain name."
msgstr ""
"L'URL du serveur distant que vous avez saisi pour la connexion n'est pas "
"valide. Vérifiez que vous avez saisi la bonne valeur. L'URL doit commencer par "
"`http://` ou `https://` suivi d'un nom de domaine."

#: minarca_client/ui/setup.py:93 minarca_client/ui/setup.py:94
msgid "Failed to connect to remote server"
msgstr "Échec de la connexion au serveur distant"

#: minarca_client/ui/setup.py:95
#, python-format
msgid ""
"Your computer cannot establish a connection to the remote server. Make sure "
"your Internet connection is working and that the following URL is accessible "
"with a Web browser: %s"
msgstr ""
"Votre ordinateur ne parvient pas à établir une connexion avec le serveur "
"distant. Assurez-vous que votre connexion Internet fonctionne et que l'URL "
"suivante est accessible avec un navigateur Web : %s"

#: minarca_client/ui/setup.py:106 minarca_client/ui/setup.py:107
msgid "Invalid username or password"
msgstr "Nom d'utilisateur ou mot de passe invalide"

#: minarca_client/ui/setup.py:108
#, python-format
msgid ""
"The username or password you entered to connect to the remote server is not "
"valid.\n"
"\n"
"Details: %s"
msgstr ""
"Le nom d'utilisateur ou le mot de passe que vous avez saisi pour vous connecter "
"au serveur distant n'est pas valide.\n"
"\n"
"Détails : %s"

#: minarca_client/ui/setup.py:120
msgid "Connection failed"
msgstr "Échec de la connexion"

#: minarca_client/ui/setup.py:121
#, python-format
msgid "Failed to establish connectivity with remote server: %s"
msgstr "Échec de l'établissement de la connectivité avec le serveur distant : %s"

#: minarca_client/ui/setup.py:130 minarca_client/ui/setup.py:131
msgid "Unknown problem when connecting to the remote server"
msgstr "Problème inconnu lors de la connexion au serveur distant"

#: minarca_client/ui/setup.py:132
#, python-format
msgid ""
"An error occurred during the connection to Minarca server.\n"
"\n"
"Details: %s"
msgstr ""
"Une erreur s'est produite lors de la connexion au serveur Minarca.\n"
"\n"
"Détails : %s"

#: minarca_client/ui/setup.py:146
msgid "Repository name already exists"
msgstr "Le nom du dépôt existe déjà"

#: minarca_client/ui/setup.py:148
msgid ""
"The repository name you have entered already exists on the remote server. If "
"you continue with this repository, you will replace it's content using this "
"computer. Otherwise, you must enter a different repository name."
msgstr ""
"Le nom de dépôt que vous avez entré existe déjà sur le serveur distant. Si vous "
"continuez avec ce nom, vous remplacerez son contenu avec celui de cet "
"ordinateur. Sinon, vous devez entrer un autre nom de dépôt."

#: minarca_client/ui/status.py:56
#, python-format
msgid "Welcome %s"
msgstr "Bienvenue %s"

#: minarca_client/ui/status.py:65
#, python-format
msgid "Backup paused until %s"
msgstr "Sauvegarde en pause jusqu'au %s"

#: minarca_client/ui/status.py:70
msgid "Backup is healthy"
msgstr "Sauvegarde saine"

#: minarca_client/ui/status.py:72
msgid "Backup failed"
msgstr "Échec de la sauvegarde"

#: minarca_client/ui/status.py:74
msgid "Backup in progress"
msgstr "Sauvegarde en cours"

#: minarca_client/ui/status.py:76
msgid "Backup is stale"
msgstr "Sauvegarde obsolète"

#: minarca_client/ui/status.py:78
msgid "Backup was interrupted"
msgstr "Sauvegarde interrompue"

#: minarca_client/ui/status.py:80
msgid "No backup yet"
msgstr "Pas encore de sauvegarde"

#: minarca_client/ui/status.py:81
msgid "Backup is not healthy"
msgstr "Sauvegarde mal en point"

#: minarca_client/ui/status.py:106
#, python-format
msgid "Complete successfully on %s."
msgstr "Terminer avec succès le %s."

#: minarca_client/ui/status.py:108
#, python-format
msgid ""
"Failed on %s\n"
"%s"
msgstr ""
"Échec le %s\n"
"%s"

#: minarca_client/ui/status.py:110
msgid "Backup is currently running in background and using system resources."
msgstr ""
"La sauvegarde s'exécute actuellement en arrière-plan et utilise les ressources "
"système."

#: minarca_client/ui/status.py:112
#, python-format
msgid ""
"Was started in background on %s, but is currently stale an may use system "
"resources."
msgstr ""
"Une sauvegarde a été démarré en arrière-plan le %s, mais elle est actuellement "
"obsolète et peut utiliser des ressources système."

#: minarca_client/ui/status.py:115
#, python-format
msgid ""
"Was interrupted on %s. May be caused by loss of connection, computer standby or "
"manual interruption."
msgstr ""
"A été interrompu le %s. Cette interruption peut être due à une perte de "
"connexion, à une mise en veille de l'ordinateur ou à une interruption manuelle."

#: minarca_client/ui/status.py:120
msgid ""
"Initial backup need to be started. You may take time to configure your "
"parameters and start your initial backup manually."
msgstr ""
"La sauvegarde initiale doit être lancée. Vous pouvez prendre le temps de "
"configurer vos paramètres et de lancer votre sauvegarde initiale manuellement."

#: minarca_client/ui/status.py:135
msgid "Stop backup"
msgstr "Arrêter la sauvegarde"

#: minarca_client/ui/status.py:136
msgid "Start backup"
msgstr "Démarrer la sauvegarde"

#: minarca_client/ui/status.py:179 minarca_client/ui/status.py:180
msgid "Fail to start backup !"
msgstr "Échec du démarrage de la sauvegarde !"

#: minarca_client/ui/status.py:181
msgid ""
"A fatal error occurred when trying to start the backup process. This usually "
"indicate a problem with the installation. Try re-installing Minarca Backup."
msgstr ""
"Une erreur fatale s'est produite lors de la tentative de démarrage du processus "
"de sauvegarde. Cela indique généralement un problème avec l'installation. "
"Essayez de réinstaller Minarca Backup."

#: minarca_client/ui/templates/credential.html:5
msgid "Enter user account information for running this tasks."
msgstr "Entrez les informations du compte utilisateur pour exécuter cette tâche."

#: minarca_client/ui/templates/credential.html:9
msgid "User name:"
msgstr "Nom de l'utilisateur :"

#: minarca_client/ui/templates/credential.html:15
msgid "Password:"
msgstr "Mot de passe :"

#: minarca_client/ui/templates/credential.html:21
msgid "OK"
msgstr "OK"

#: minarca_client/ui/templates/credential.html:22
msgid "Cancel"
msgstr "Cancel"

#: minarca_client/ui/templates/home.html:7
msgid "Home"
msgstr "Accueil"

#: minarca_client/ui/templates/home.html:10
#: minarca_client/ui/templates/patterns.html:3
msgid "Select files"
msgstr "Sélection de fichiers"

#: minarca_client/ui/templates/home.html:14
#: minarca_client/ui/templates/schedule.html:4
msgid "Schedule backup"
msgstr "Programmer la sauvegarde"

#: minarca_client/ui/templates/home.html:18
#: minarca_client/ui/templates/settings.html:4
msgid "Settings"
msgstr "Paramètres"

#: minarca_client/ui/templates/home.html:22
msgid "Help"
msgstr "Aide"

#: minarca_client/ui/templates/patterns.html:6
msgid "Add file"
msgstr "Ajouter fichier"

#: minarca_client/ui/templates/patterns.html:7
msgid "Add folder"
msgstr "Ajouter dossier"

#: minarca_client/ui/templates/patterns.html:8
msgid "Add ..."
msgstr "Ajouter ..."

#: minarca_client/ui/templates/patterns.html:14
msgid "Remove pattern"
msgstr "Supprimer"

#: minarca_client/ui/templates/schedule.html:7
msgid ""
"Configure when you want your backup to take place, or suspend it temporarily."
msgstr ""
"Configurez le moment où vous souhaitez que votre sauvegarde ait lieu, ou "
"suspendez-la temporairement."

#: minarca_client/ui/templates/schedule.html:14
msgid "Frequency"
msgstr "Fréquence"

#: minarca_client/ui/templates/schedule.html:15
msgid "Hourly"
msgstr "Horaire"

#: minarca_client/ui/templates/schedule.html:16
msgid "Once a day"
msgstr "Une fois par jour"

#: minarca_client/ui/templates/schedule.html:17
msgid "Twice a day"
msgstr "Deux fois par jour"

#: minarca_client/ui/templates/schedule.html:23
msgid "Suspend"
msgstr "Suspendre"

#: minarca_client/ui/templates/schedule.html:25
msgid "Suspended"
msgstr "Suspendu"

#: minarca_client/ui/templates/schedule.html:28
msgid ""
"This feature lets you pause backup operations for 24 hours. Ideal when you need "
"to reduce the use of system resources or free up bandwidth for other activities."
msgstr ""
"Cette fonction vous permet d'interrompre les opérations de sauvegarde pendant "
"24 heures. Elle est idéale pour réduire l'utilisation des ressources du système "
"ou libérer de la bande passante pour d'autres activités."

#: minarca_client/ui/templates/schedule.html:34
msgid "Advance Options"
msgstr "Options avancées"

#: minarca_client/ui/templates/schedule.html:35
msgid "Run whether user is logged on or not"
msgstr "Exécuter si l'utilisateur est connecté ou non"

#: minarca_client/ui/templates/schedule.html:38
msgid ""
"This feature allows Minarca to run scheduled backup tasks regardless of whether "
"a user is currently logged on to the system or not. This is particularly useful "
"for server."
msgstr ""
"Cette fonction permet à Minarca d'exécuter des tâches de sauvegarde "
"programmées, qu'un utilisateur soit ou non connecté au système. Cette fonction "
"est particulièrement utile pour les serveurs."

#: minarca_client/ui/templates/settings.html:9
msgid "Updates"
msgstr "Mise à jour"

#: minarca_client/ui/templates/settings.html:10
msgid "Check now"
msgstr "Vérifier maintenant"

#: minarca_client/ui/templates/settings.html:14
msgid "Check for update at startup"
msgstr "Vérifier les mise à jour au démarrage"

#: minarca_client/ui/templates/settings.html:18
msgid "Checking for updates"
msgstr "Recherche de mises à jour"

#: minarca_client/ui/templates/settings.html:20
msgid "Current version is up-to-date"
msgstr "La version actuelle est à jour"

#: minarca_client/ui/templates/settings.html:22
msgid "Your version may be upgraded"
msgstr "Mise à jour disponible"

#: minarca_client/ui/templates/settings.html:24
msgid "Cannot check for updates. Try again later."
msgstr "Impossible de rechercher des mises à jour. Réessayez plus tard."

#: minarca_client/ui/templates/settings.html:36
msgid "Disconnect the backup"
msgstr "Arrêter la sauvegarde"

#: minarca_client/ui/templates/settings.html:37
msgid "Disconnect"
msgstr "Arrêter"

#: minarca_client/ui/templates/settings.html:39
msgid "When you disconnect this computer, the backup process will no longer work."
msgstr ""
"Lorsque vous débranchez cet ordinateur, le processus de sauvegarde ne "
"fonctionne plus."

#: minarca_client/ui/templates/setup.html:5
msgid "Data Backup"
msgstr "Sauvegarde de données"

#: minarca_client/ui/templates/setup.html:10
msgid "Log me in"
msgstr "Me connecter"

#: minarca_client/ui/templates/setup.html:16
msgid "Remote Minarca server"
msgstr "Serveur Minarca distant"

#: minarca_client/ui/templates/setup.html:19
msgid "e.g.: https://www.servername.com"
msgstr "p.e.: https://www.servername.com/"

#: minarca_client/ui/templates/setup.html:22
msgid "Username"
msgstr "Nom d'utilisateur"

#: minarca_client/ui/templates/setup.html:27
msgid "Password"
msgstr "Mot de passe"

#: minarca_client/ui/templates/setup.html:30
msgid ""
"If two-factor authentication is enabled on your account, use an Access Token."
msgstr ""
"Si l'authentification à deux facteurs est activée sur votre compte, utilisez un "
"jeton d'accès."

#: minarca_client/ui/templates/setup.html:34
msgid "Repository name"
msgstr "Nom du dépôt"

#: minarca_client/ui/templates/setup.html:40
msgid "Sign in"
msgstr "Se connecter"

#: minarca_client/ui/templates/setup.html:43
msgid "Connecting"
msgstr "Connexion"

#: minarca_client/ui/templates/setup.html:47
msgid ""
"To configure Minarca to backup your data, you must provide URL to Minarca "
"server. This information should be provided by your system administrator."
msgstr ""
"Pour configurer Minarca pour sauvegarder vos données, vous devez fournir l'URL "
"d'un serveur Minara. Ces informations doivent être fournies par votre "
"administrateur système."

#: minarca_client/ui/templates/status.html:7
msgid "View my account online"
msgstr "Voir mon compte en ligne"

#: minarca_client/ui/templates/status.html:18
msgid "Backup Status: "
msgstr "État de la sauvegarde : "

#: minarca_client/ui/templates/status.html:26
msgid "Last Backup: "
msgstr "Dernière sauvegarde : "

#~ msgid "a password is required"
#~ msgstr "un mot de passe est requis"
